com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs

```diff
@@ -1678,6 +1678,13 @@ Matrix4x4 GetJitteredProjectionMatrix(Matrix4x4 origProj)
                 taaJitter = Vector4.zero;
                 return origProj;
             }
+            #if UNITY_2021_2_OR_NEWER
```

Comment:
Don't think we need that #if here - @sebastienlagarde please correct me if I'm wrong. 

Did the new API land in a19 or later? I ask because we're part of trunk right now and we have the minimum version (the package.json file) here set to 2021.2.0a19.

Reply:
Ah I wasn't sure how it goes these days; found 12 other cases of `UNITY_2021_2_OR_NEWER` in the Graphics code and assumed it's still needed :)

Reply:
The API landed in a19

Reply:
Hey, yes the define is not needed, safe to remove it but also doesn't harm if someone see the fix and think to backport it, so let's let it here.